portal-impl/src/com/liferay/portal/service/persistence/impl/RoleFinderImpl.java

```diff
@@ -723,6 +771,66 @@ public Role findByC_N(long companyId, String name)
 		}
 	}
 
+	protected int doCountByGroupRoleAndTeamRole(
+		long companyId, String[] excludedNames, int[] types,
```

Comment:
JournalArticleFinderImpl has multiple methods that take in a list. I wanted you to use a list because the logic would be much simpler in the JSP - https://github.com/jonathanmccann/liferay-portal/pull/1153/commits/e2f67f20f8f79d25f428cd741a41e30260a62da7#diff-513a3cfacd5a0b5b6bf29636da5fbe0eR293